termwiz/src/escape/osc.rs

```diff
@@ -384,6 +387,7 @@ osc_entries!(
     SetCurrentWorkingDirectory = "7",
     /// See https://gist.github.com/egmontkob/eb114294efbcd5adb1944c9f3cb5feda
     SetHyperlink = "8",
+    SetHyperfile = "88",
```

Comment:
this uses OSC 88, but other places still use and refer to OSC 8

Reply:
Here I used 8 at first. But I noticed that this specific line when having hyperfile as 8 and hyperlink as 8, the tests will fail. Because the hyperlink would receive a hyperfile, when the hyperfile=8 is assigned secondly, or if I switch places. And if I place hyperlink = 8 secondly then this would make the hyperfile test fail. So I added a random number 88, I don't know now what OSC 8 is, but I observed that 88 number fixed the interaction.

Reply:
Do you recon we could use another number besides 8 ?

Reply:
I'd suggest going with 88 as a thought experiment to see this working, but overall: I don't think we should be adding a new OSC code for this when we can use hyperlink rules with the existing hyperlinks OSC 8 :)